manifests/j/JumpingBytes/PureSync/7.1.2/JumpingBytes.PureSync.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: JumpingBytes.PureSync

```

```suggestion
PackageIdentifier: JumpingBytes.PureSync.Personal
```
As the package has two editions, a personal and a professional one, I would prefer if we disambiguate the two in the PackageIdentifier.